app/src/main/java/com/uc/mymovies/dao/UserDAO.kt

```diff
@@ -1,5 +1,8 @@
 package com.uc.mymovies.dao
 
+import androidx.room.Dao
+
+@Dao
```

Comment:
Is this used in Room?  Verify that first.

If so, and if  you add @Dao, do you want to add the annotations above the methods themselves?